### PR TITLE
[WINC-921]: fast-forward community-4.13 branch

### PR DESCRIPTION
**Pull request by alinaryan**
This PR adds the ability to fast-forward commits from master to community-4.13. This will prevent us from
having to manually backport PRs to the community branch. Fast forwards happen from master as opposed to release branches since the latest community tracks the master branch. The DESTINATION_BRANCH will need to be updated as part of the
branching day tasks. 

Once we ensure this works for the master branch, we will open a follow-up PR to fast-forward the other release branches to their equivalent community branches.

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml
Comment (aravindhp) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:177]: Please mention why you are doing this for master instead of community-4.13.

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.11.yaml
Comment (aravindhp) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.11.yaml:156]: I suggest doing this just for master ensure it works before adding to the other branches.

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for doing this @alinaryan. Please rename the PR and commit to `fast-forward community-4.13 branch`

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.11.yaml
Comment (sebsoto) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.11.yaml]: Please update the PR description as well

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml
Comment (alinaryan) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:177]: I couldn't add comments, I added this to the commit and PR description.

Comment (aravindhp) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:177]: Yes that is where I meant for it to no. No need for inline comments.

**Review by aravindhp — APPROVED** [review]

**Comment by aravindhp** [comment]
/hold

for other folks to review

**Comment by mtnbikenc** [comment]
LGTM
I want this to merge before we merge more PRs to WMCO master.
/hold cancel

**Pull request merged**
